lib/drab/element.ex

```diff
@@ -288,6 +288,29 @@ defmodule Drab.Element do
     set_prop!(socket, selector, %{"style" => Map.new(properties)})
   end
 
+  @doc """
+  Broadcasting version of `set_style/3`.
+
+  It does exactly the same as `set_style/3`, but instead of pushing the message to the current
+  browser, it broadcasts it to all connected users.
+
+  Always returns `{:ok, :broadcasted}`.
+
+  See `Drab.Core.broadcast_js/2` for broadcasting options.  
+  """
+  @spec broadcast_style(Drab.Core.subject(), String.t(), map | Keyword.t()) :: Drab.Core.result()
```

Comment:
Return of broadcasting functions should be type of `Drab.Core.bcast_result()`

Reply:
ah, yes! Damned copy/past..